scripts/generate-loggregator-certs

```diff
@@ -1,65 +1,72 @@
+    bbs_ca_name="$ca_name"
 else
-    bbs_ca_cert_path=$(readlink ${1})
-    bbs_ca_key_path=$(readlink ${2})
-    bbs_ca_name=$(basename ${bbs_ca_cert_path} ".crt")
-    ln -s ${bbs_ca_cert_path} ${depot_path}/
-    ln -s ${bbs_ca_key_path} ${depot_path}/
+    bbs_ca_cert_path="$(find "$1" -name 'cf-diego-ca.crt')"
+    bbs_ca_key_path="$(find "$1" -name 'cf-diego-ca.key')"
+    bbs_ca_name="cf-diego-ca"
+    if [[ ! -f "$bbs_ca_cert_path" || ! -f "$bbs_ca_key_path" ]]; then
+        echo -e "\e[1m\e[33m[ERROR] Please execute generate-cf-diego-certs before running this script.\e[0m"
+        exit 1
+    fi
+    ln -s "${bbs_ca_cert_path}" "${depot_path}"/
+    ln -s "${bbs_ca_key_path}" "${depot_path}"/
 fi
 
-# Install certstrap
-go get -v github.com/square/certstrap
-
-# CA to distribute to loggregator certs
-certstrap --depot-path ${depot_path} init --passphrase '' --common-name loggregatorCA
-mv -f ${depot_path}/loggregatorCA.crt ${depot_path}/loggregator-ca.crt
-mv -f ${depot_path}/loggregatorCA.key ${depot_path}/loggregator-ca.key
-mv -f ${depot_path}/loggregatorCA.crl ${depot_path}/loggregator-ca.crl
+if [[ "$2" == "--recreate-ca" ]] || [[ -z "$2" && ! -f "$depot_path/$ca_name.crt" ]]; then
+  # CA to distribute to loggregator certs
+  certstrap --depot-path "${depot_path}" init --passphrase '' --common-name loggregatorCA
+  mv -f "${depot_path}/loggregatorCA.crt" "${depot_path}/$ca_name.crt"
+  mv -f "${depot_path}/loggregatorCA.key" "${depot_path}/$ca_name.key"
+  mv -f "${depot_path}/loggregatorCA.crl" "${depot_path}/$ca_name.crl"
+elif [[ -z "$2" && -f "$depot_path/$ca_name.crt" ]]; then
+  echo -e "\e[1m\e[33m[INFO] Using existing loggregator CA, recreating only client/server certificates\e[0m"
+fi
+find "${depot_path}" -type f -not -name "$ca_name*" -delete
 
 # Doppler certificate
-certstrap --depot-path ${depot_path} request-cert --passphrase '' --common-name doppler
-certstrap --depot-path ${depot_path} sign doppler --CA loggregator-ca
+certstrap --depot-path "${depot_path}" request-cert --passphrase '' --common-name doppler
+certstrap --depot-path "${depot_path}" sign doppler --CA "$ca_name"
 
 # Traffic Controller certificate
-certstrap --depot-path ${depot_path} request-cert --passphrase '' --common-name trafficcontroller
-certstrap --depot-path ${depot_path} sign trafficcontroller --CA loggregator-ca
+certstrap --depot-path "${depot_path}" request-cert --passphrase '' --common-name trafficcontroller
+certstrap --depot-path "${depot_path}" sign trafficcontroller --CA "$ca_name"
 
 # Traffic Controller<->CAPI mutual TLS certificate
-certstrap --depot-path ${depot_path} request-cert --passphrase '' --common-name cc_trafficcontroller
-certstrap --depot-path ${depot_path} sign cc_trafficcontroller --CA ${bbs_ca_name}
+certstrap --depot-path "${depot_path}" request-cert --passphrase '' --common-name cc_trafficcontroller
+certstrap --depot-path "${depot_path}" sign cc_trafficcontroller --CA "${bbs_ca_name}"
```

Comment:
When I tried to run `./scripts/generate-cf-certs .`, it failed on this line consistently. It looks like `generate-loggregator-certs` is failing to find the Diego BBS CA (`cf-diego-ca`):
```
stat /Users/davidanthonysabeti/workspace/cf-release/loggregator-certs/cf-diego-ca.crt: no such file or directory
```

(I think it expects the argument to `--CA` to be the _path_ to the CA file. Before this refactoring, the user was required to copy that CA cert into the `loggregator-certs` directory. Now that we're scripting it, `generate-cf-certs` will probably need to find a way to copy that file to the correct location.)

Reply:
My bad, I didn't consider the `.` case :) I was testing it with `$PWD`. Unfortunately, certstrap accepts only abs path and not rel path, this is way it's failing if you pass `.`
I will fix it, good catch 👍  

What the script actually does is to symlink the diego CA and CAkey to the current dir, so certstrap can sign the cert. This is way the copying is no longer needed.